src/components/Country.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropsType from 'prop-types';
+import CovidVirus from './CovidVirus';
+
+const Country = ({ country, handleClick }) => (
+  <div
+    onClick={() => handleClick(country.links[0].href, country.id)}
+    onKeyDown={(e) => e.key === 'Enter' && handleClick(country.links[0].href, country.id)}
+    className="country-div country-div2 country-div3 country-odd-div country-even-div"
+    key={country.id}
+    role="button"
```

Comment:
- [ ] As requested in the previous review, kindly implement a redux filter/ search functionality on your Homepage so as to enable filtering of countries.

Reply:
Filter function was implemented in the CountriesList component which is the home page.
![filter](https://user-images.githubusercontent.com/33420420/142738426-92792cc1-a41c-48f8-a3b1-843fe26888cd.png)

Reply:
Please this is my last review. :pray: